### PR TITLE
third pr jesulayomi

### PR DESCRIPTION
**Pull request by Layomii**
## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

File: docs/TEAM.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
+| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|                                                                                                                                |
```
Comment (RafaelBatman55) [inline docs/TEAM.md:6]: I think it's something wrong with this change. The table totally broke.

**Comment by RafaelBatman55** [comment]
Next time try not to create a PR with different documents without a connection.

It is hard for reviewers.

File: docs/TEAM.md [hunk 2]
```diff
@@ -30,8 +30,8 @@
...
-|                                                                                          | [Neaz Mahmood](https://github.com/Neaz-Mahmood)              | I am from Dhaka, Bangladesh. I am a second year Computer Science undergraduate. I know basic html, css, javascript. I am here to learn & contribute to 'twindle'. This is my first open source project.                                                                                                                                                                                                    |
+|                                                                                          | [Maria Rivera](https://github.com/GorettiRivera)             | I have basic knowledge of Javascript, HTML and CSS. Currently working in different open source projects
```
Comment (RafaelBatman55) [inline docs/TEAM.md:33]: Please add again the final "|".

File: docs/TEAM.md [hunk 3]
```diff
@@ -40,7 +40,8 @@
...
 | | [OmoloG](https://github.com/geowen74) | From Nairobi, Kenya. Have Knowledge on HTML, CSS and some basic on javascript and Mysql. Aspiring to be a full-stack developer. Currently learning javascript.
-| <img src="./images/team/shekharranjan.jpg" width="90px" height="90px"> | [Shekhar Ranjan](https://github.com/shekhar10feb)                   | I'm from India, have basic knowledge of Java, SQL, HTML, CSS and still learning JavaScript, want to learn more and explore advance stuffs.|
+| <img src="./images/team/shekharranjan.jpg" width="90px" height="90px"> | [Shekhar Ranjan](https://github.com/shekhar10feb)                   | I'm from India, have basic knowledge of Java, SQL, HTML, CSS and still learning JavaScript, want to learn more and explore advance stuffs.
```
Comment (RafaelBatman55) [inline docs/TEAM.md:43]: Please add again the final "|".

File: docs/TEAM.md [hunk 4]
```diff
@@ -40,7 +40,8 @@
...
 | | [OmoloG](https://github.com/geowen74) | From Nairobi, Kenya. Have Knowledge on HTML, CSS and some basic on javascript and Mysql. Aspiring to be a full-stack developer. Currently learning javascript.
-| <img src="./images/team/shekharranjan.jpg" width="90px" height="90px"> | [Shekhar Ranjan](https://github.com/shekhar10feb)                   | I'm from India, have basic knowledge of Java, SQL, HTML, CSS and still learning JavaScript, want to learn more and explore advance stuffs.|
+| <img src="./images/team/shekharranjan.jpg" width="90px" height="90px"> | [Shekhar Ranjan](https://github.com/shekhar10feb)                   | I'm from India, have basic knowledge of Java, SQL, HTML, CSS and still learning JavaScript, want to learn more and explore advance stuffs.                                                                                                               
+|                                                                                          | [Adetola Jesulayomi](https://github.com/Layomii)               | My name is Adetola Jesulayomi, I have a little bit of experience in HTML, CSS and Python
```
Comment (RafaelBatman55) [inline docs/TEAM.md:44]: Please add again the final "|".

**Review by RafaelBatman55 — CHANGES_REQUESTED** [review]
Please check again this PR. You tried to delete some important documents.

Another issue:

- In "firstpr/adetola_jesulayomi..md", you need to remove double "." before md extension.

**Comment by proful** [comment]
Too many unwanted changes

**Comment by johnjacobkenny** [comment]
@RafaelBatman55 would you like to help @Layomii create a new PR?

**Pull request closed without merging**